frontend/src/assets/css/table.css

```diff
@@ -799,6 +799,10 @@ $indent-collapsible-offset-lg: 24px;
     border-left: 1px solid #afbfca;
 }
 
+.table-cell-label-container {
```

Comment:
can we document that this one it has to be in sync with Constants.TBL_CELL_LABEL_MAX ?

Reply:
sure. good point. ty